resspect/fit_lightcurves.py

```diff
@@ -174,217 +213,52 @@ def load_snpcc_lc(self, path_to_data: str):
         path_to_data: str
             Path to text file with data from a single SN.
         """
-
-        # set the designation of the data set
         self.dataset_name = 'SNPCC'
-
-        # set filters
         self.filters = ['g', 'r', 'i', 'z']
 
-        # set SN types
-        snii = ['2', '3', '4', '12', '15', '17', '19', '20', '21', '24', '25',
-                '26', '27', '30', '31', '32', '33', '34', '35', '36', '37',
-                '38', '39', '40', '41', '42', '43', '44']
+        lc_data = np.array(read_file(path_to_data), dtype=object)
+        photometry_raw, header = self._iterate_snpcc_lc_data(lc_data)
 
-        snibc = ['1', '5', '6', '7', '8', '9', '10', '11', '13', '14', '16',
-                 '18', '22', '23', '29', '45', '28']
+        if photometry_raw.size > 0:
```

Comment:
more succint: `if photometry_raw:`

Reply:
It will not work since `photometry_raw` is a np.array

Reply:
ah, fair enough!